src/components/character.css

```
@@ -32,3 +32,26 @@
 .characters > * {
   margin: 10px;
 }
+.favourite__img {
+  height: 50px;
+  width: 50px;
+}
+.favourite__img:hover {
+  color: white;
+}
+
+.favourite__btn {
+  background: none;
+  cursor: pointer;
+  border: none;
+  margin-top: 10px;
+}
+.favourite__btn:hover {
+  border: 1px solid white;
+}
```

On hover the button shifts the whole layout a bit.
You could use an outline so it doesn't shift the layout.
```suggestion
.favourite__btn:hover {
  outline: 1px solid white;
}
```

---

Thanks a lot!